### PR TITLE
Fiori ngx for details views (improvement)

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Get rid of `sf-*` classes. Replace those elements with Fundamental-NGX (preferred) or `.fd-*` classes.
I am aware of leaving one `sf-*` class in `environment-details.component.html:82`. I couldn't get the same effect using any of above methods.

UI tests are failing because they're also failing on _master_.

**Related issue(s)**

### CODE REVIEW

File: core/src/app/content/settings/organisation/organisation.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="sf-section">
+<div class="fd-section">
```
Comment (jesusreal) [inline core/src/app/content/settings/organisation/organisation.component.html]: There are some custom classes used here and declared as css selectors in `organisation.component.scss`. Even though they don't have the `sf-` prefix, they are still custom styles. Can you please try to replace them with fundamental classes and remove the custom styles?

File: core/src/app/content/settings/organisation/organisation.component.ts [hunk 2]
```diff
@@ -6,8 +6,7 @@ import * as FileSaver from 'file-saver';
 @Component({
   selector: 'app-organisation',
   templateUrl: './organisation.component.html',
-  styleUrls: ['./organisation.component.scss'],
-  host: { class: 'sf-content' }
+  styleUrls: ['./organisation.component.scss']
```
Comment (jesusreal) [inline core/src/app/content/settings/organisation/organisation.component.ts]: In this file there is an override for `.fd-container--flex`. Please remove the override in the scss file and in the html replace the custom class `direction--column` with for example `fd-has-float-right`: https://sap.github.io/fundamental/resources/design-system-utilities.html

File: core/src/app/content/environments/environment-details/environment-details.component.html [hunk 3]
```diff
@@ -1,94 +1,92 @@
...
-                <a class="sf-sku__label" (click)="navigateToServices()">
-                  {{ services && services.length === 1 ? 'Service' : 'Services' }}
-                </a>
-              </div>
+        </div>
+      </fd-panel-body>
+    </fd-panel>
+    <fd-panel>
+      <fd-panel-body>
+        <div class="row">
```
[the remark this answers is not in the document]
Comment (parostatkiem) [reply core/src/app/content/environments/environment-details/environment-details.component.html]: No. Fundamentau's docu doesn't say anything about such classes so I assume they're not present. Fundamental-ngx provides columns-like layout but it doesn't work or look good IMO. That's why I left these.

File: core/src/app/content/environments/environment-details/environment-details.component.html [hunk 4]
```diff
@@ -1,94 +1,92 @@
...
+          <div class="col">
+            <div class="text-center">
```
[the remark this answers is not in the document]
Comment (parostatkiem) [reply core/src/app/content/environments/environment-details/environment-details.component.html]: No

**Comment by parostatkiem** [comment]
Thanks @jesusreal for discovering _Design System Utilities_ 😂

**Pull request merged**